### DISCUSSION TITLE
jmxOptions: {} not working

### DISCUSSION THREAD

**Discussion by Veljen** [strimzi/strimzi-kafka-operator#9571] [General]
Hi,

I have added the jmxOptions: {} and I am geeting empty server reply from 9999 port  (localhost).  jmxprometheusmetrics works fine with port 9404 but not this one.  I am using latest operator version 0.39.   

I am not seeing any relevant logs in broker and operator for this jmxoptions.  Kindly help me 
how to setup this and troubleshoot further.

[kafka@poc-strimizi-kafka-0 kafka]$ curl http:/poc-strimizi-kafka-0.poc-strimizi-kafka-brokers.strimzi-kafka-1.svc.cluster.local:9999
curl: (52) Empty reply from server

**Comment by scholzj** [comment]
JMX is a protocol on its own. It is not HTTP. So you are not expected to get anything with HTTP client such as `curl`. You have to use some JMX client to connect to it.
